pwa/pages/_app.tsx

```diff
@@ -12,7 +12,8 @@ type AppPropsWithLayout = AppProps & {
 };
 
 function MyApp({Component, pageProps, ...props}: AppPropsWithLayout) {
-  const googleClientId = process.env.GOOGLE_CLIENT_ID;
+  const googleClientId =
```

Comment:
I think is better to keep the old version

Reply:
I forgot this part, it was necessary to make google agenda synchronization working good